src/verifier.js

```diff
@@ -235,22 +235,22 @@ export default class Verifier {
     let localHash = await this._doAsyncAction(
       SUB_STEPS.computeLocalHash,
       async () =>
-        checks.computeLocalHash(this.documentToVerify, this.version)
+        inspectors.computeLocalHash(this.documentToVerify, this.version)
     );
 
     // Compare hashes
     this._doAction(SUB_STEPS.compareHashes, () =>
-      checks.ensureHashesEqual(localHash, this.receipt.targetHash)
+      inspectors.ensureHashesEqual(localHash, this.receipt.targetHash)
     );
 
     // Check receipt
     this._doAction(SUB_STEPS.checkReceipt, () =>
-      checks.ensureValidReceipt(this.receipt)
+      inspectors.ensureValidReceipt(this.receipt)
     );
 
     // Check expiration date
     this._doAction(SUB_STEPS.checkExpiresDate, () =>
-      checks.ensureNotExpired(this.expires)
+      inspectors.ensureNotExpired(this.expires)
```

Comment:
I think ultimately something that could be cool to have would be some sort of map of the substep and its associated method, especially more per process. Then we would only be maintaining a map and not code.